core/arch/arm/kernel/boot.c

```diff
@@ -1358,27 +1366,31 @@ unsigned long __weak get_aslr_seed(void *fdt)
 
 	if (rc) {
 		DMSG("Bad fdt: %d", rc);
-		return 0;
+		goto err;
```

Comment:
I don't think it's meaningful with a fallback. Either are we're picking up the seed from DT or we have our own way, not point in both.

Reply:
What if OP-TEE is built with FDT support, but some platform revision does not provide it? This is sort of my case: some builds of ARM TF provide FDT, some - are not. Probably, there can be similar platforms.

It is debatable what should have higher priority - seed from FDT or `plat_` - provided one.

Reply:
OK, if it's useful.